opentelemetry-api/src/opentelemetry/baggage/propagation/__init__.py

```diff
@@ -68,6 +65,9 @@ def extract(
                 urllib.parse.unquote(value).strip(),
                 context=context,
             )
+            total_baggage_entries -= 1
```

Comment:
Curious as to why we are counting backwards as opposed to simply just counting to `_MAX_PAIRS`?

Reply:
:shrug: It was implemented like that, so I kept the implementation.